README.md

```diff
@@ -24,14 +24,16 @@ You can use this crate from cargo with
 
 ```toml
     [dependencies]
-    cassandra-cpp-sys = "0.8"
+    cassandra-cpp-sys = "0.10"
 ```
 
 ## Important note regarding version 0.9.0
 
-This branch is version 0.8.x of cassandra-cpp-sys. Development has begun on version 0.9.x using a newer version of bindgen,
-but it is not yet ready for use. See the [version-0.9](https://github.com/Metaswitch/cassandra-sys-rs/tree/version-0.9) branch
-to follow its development. In due course this will be merged to master, but for the moment we recommend you use version 0.8.x.
+At one point some development was done on preparing a version 0.9.0 
+using a newer version of bindgen, but the work was not completed or released. See the 
+[version-0.9](https://github.com/Metaswitch/cassandra-sys-rs/tree/version-0.9) branch
```

Comment:
Yes - it was pushed to crates.io as version 0.9, though as far as I can see no one ever depended on it publicly. I want to leave the door open to merging the crates again one day, so we need to treat 0.9 as reserved. This explains why, and where it went if anyone is wondering.